spec/unit/shoulda/matchers/action_controller/permit_matcher_spec.rb

```diff
@@ -205,25 +293,31 @@
           for(:create)
       end
 
-      expect(&assertion).to fail_with_message(
-        'Expected controller to permit city and country, but it did not.'
-      )
+      message =
+        "Expected POST #create to restrict parameters to " +
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.